nemo/collections/nlp/models/text_classification/text_classification_model.py

```diff
@@ -159,31 +191,36 @@ def _setup_tokenizer(self, cfg: DictConfig):
         self.tokenizer = tokenizer
```

Comment:
Please add `register_artifact` to save the tokenizer model if available

Reply:
But do we have the process to load the tokenizer or I should add that too? Just saving the tokenizer would not work?

Reply:
Added that one, but how to specify to load the tokenizer from the artifacts when model is loaded. It looks like that we just save the file and do not use it. Have you tried deleting the vocab file from the system and load a model from nemo?